tests/Unit/DataStructures/Tensor/Expressions/Test_Product.cpp

```diff
@@ -33,6 +33,53 @@ void assign_unique_values_to_tensor(
   }
 }
 
+/// \ingroup TestingFrameworkGroup
```

Comment:
This is in a cpp file, so don't need/use a Doxygen comment (two slashes, no `\ingroup`). I'm generally fine with formatting the comment in the Doxygen style since that is actually reasonably easy to read compared to when everyone comes up with their own comment style :P

This is true for the other functions in this file (clearly I missed that in other PRs), so please just clean that up in a separate PR

Reply:
Oh I didn't realize that doxygen documentation wasn't generated for cpp files (clearly, I haven't been thorough enough in checking all of my documentation, which is on me). So I'm clear, you're saying you're fine with even keeping the doxygen commands/labels like `\brief`, `\details`, `tparam`, `\param`, `return`?

Reply:
Yes, that's right